### PR TITLE
Update demo notebook 02.

### PR DESCRIPTION
**Pull request by jjbrosnan**
This won't be fully ready for merge until Jianfeng adds column formatting to V2 Python source

### CODE REVIEW

File: demo/web/src/main/notebooks/02 Stream and Batch Together.md [hunk 1]
```diff
@@ -12,10 +12,10 @@ Below you’ll do calculations and aggregations on stream and batch data using i
 First, hook up a Kafka stream. (This is the same script from the first notebook.) Our [how-to guide](https://deephaven.io/core/docs/how-to-guides/kafka-stream/) provides detail on the integration.
 
 ```python
-from deephaven import ConsumeKafka as ck
+from deephaven import kafka_consumer as ck
```
Comment (hythloda) [inline demo/web/src/main/notebooks/02 Stream and Batch Together.md:15]:
```suggestion
from deephaven import kafka_consumer as ck
from deephaven.stream.kafka.consumer import TableType, KeyValueSpec
from deephaven import Types as dht
```

**Review by hythloda — COMMENTED** [review]
I can't change non-edited lines but 22-25 need to be:
```python
        key_spec = KeyValueSpec.IGNORE,
        value_spec = ck.avro_spec('io.deephaven.crypto.kafka.TradesTopic-io.deephaven.crypto.Trade'),
        offsets=ck.ALL_PARTITIONS_SEEK_TO_END,
        table_type= TableType.Append)
```

File: demo/web/src/main/notebooks/02 Stream and Batch Together.md [hunk 2]
```diff
@@ -142,11 +142,11 @@ You can use Deephaven for **time-series and relational** joins on static tables,
 Please note that you do not have to think about whether a named_table happens to be updating (stream-like) or static (batch-like). The methods are common between the two.
 
 ```python
-join_stream_batch = agg_streaming.renameColumns("Price_Streaming = Price", "Avg_Price_Streaming = Avg_Price")\
-  .naturalJoin(agg_batch, "Exchange, Instrument", "Date_Batch = Date, Price_Batch = Price, Avg_Price_Batch = Avg_Price")\
+join_stream_batch = agg_streaming.rename_columns(["Price_Streaming = Price", "Avg_Price_Streaming = Avg_Price"])\
+  .natural_join(right=agg_batch, on=["Exchange, Instrument", "Date_Batch = Date, Price_Batch = Price, Avg_Price_Batch = Avg_Price"])\
   .formatColumns("Date = `IVORY`")\
```
Comment (hythloda) [inline demo/web/src/main/notebooks/02 Stream and Batch Together.md:147]: Check that formatColumns works.  Last I checked it was not wrapped or I missed it

Comment (jjbrosnan) [reply demo/web/src/main/notebooks/02 Stream and Batch Together.md:147]: It is not currently wrapped.  Jianfeng is working on a V2 wrapper now.  When that's merged into main, I'll update these lines and this notebook will be ready for a merge.

**Review by hythloda — APPROVED** [review]

**Pull request closed without merging**